packages/ui/src/Card.module.scss

```
@@ -0,0 +1,39 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-content: stretch;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+
+  &.typeSecondary {
+    background: none;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    box-shadow: none;
+  }
+
+  &.typeHighlighter {
+    background: c.$colorInfoLight;
+    box-shadow: none;
+  }
+
+  .heading {
+    display: flex;
+    align-items: center;
+    padding: c.$grid * 6 c.$grid * 6 0;
+    margin: 0;
+
+    .icon {
+      display: flex;
+      align-items: center;
+      margin-left: c.$grid * 2;
```

nit: redundant styles
```suggestion
      margin-left: c.$grid * 2;
```